.github/workflows/tests.yml

```
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
```

I don't think we want to maintain python 3.7, what do you think @carlio ?

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```